### PR TITLE
Update office-hours config to logical AND

### PR DESCRIPTION
**Pull request by joakimen**
https://liflig.atlassian.net/browse/CALS-679

## Description

Change schedule for office-hours config from logical OR to logical AND.

## Changes

- Change office-hours schedule predicate
- Minor refactor on `validate.sh`-script and add doc explaining local usage
- Add husky + commitlint for local dev

Added commitlint even if we don't use semantic release for this lib just because it's a nice standard to follow. If you prefer to not include it here, let me know and I will drop the husky-commit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,14 +2,20 @@
...
+    "prepare": "husky install"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/capraconsulting/renovate-config.git"
   },
   "author": "Capra Consulting",
   "license": "Apache 2.0",
   "devDependencies": {
+    "@commitlint/cli": "^18.4.4",
```
Comment (stekern) [inline package.json:15]: Minor thing in this repository, especially as its just dev dependencies, but we generally prefer to pin all dependencies to exact versions

**Review by stekern — APPROVED** [review]
Looks good.

We don't really need to enforce conventional commits here, but I don't see an issue with including it as long as it doesn't introduce a lot of maintenance overhead.

I suggest we merge this in tomorrow as the updates are effectively released after this PR has been merged and landed on trunk.

File: package.json (hunk 1 above)
Comment (joakimen) [reply package.json:15]: Yeah, I don't really have any strong opinions on this, I'll pin it

Comment (joakimen) [reply package.json]: https://github.com/capralifecycle/renovate-config/pull/41/commits/4c1f8c582fd5f6d2804ab54bf1948c866f3b080d

**Comment by joakimen** [comment]
Will hold on with merging until tomorrow. 👍🏻

**Pull request merged**